content/docs/getting-started.md

```diff
@@ -36,18 +36,16 @@ React has been designed from the start for gradual adoption, and **you can use a
 
 ### Online Playgrounds
 
-If you're interested in playing around with React, you can use an online code playground. Try a Hello World template on [CodePen](codepen://hello-world) or [CodeSandbox](https://codesandbox.io/s/new).
+To start playing with React instantly, you can use an online code playground, without having to bother configuration and environment setup. Try a Hello World template on [CodePen](codepen://hello-world) or [CodeSandbox](https://codesandbox.io/s/new).
 
 If you prefer to use your own text editor, you can also [download this HTML file](https://raw.githubusercontent.com/reactjs/reactjs.org/master/static/html/single-file-example.html), edit it, and open it from the local filesystem in your browser. It does a slow runtime code transformation, so we'd only recommend using this for simple demos.
 
-### Add React to a Website
+### Through the `<script>` Tag
```

Comment:
`Add React to a Website` doesn't tell me the real thing that we want to address in this section. We want to tell people all you need is a script tag.

Reply:
I see what you want to do but I'm not sure "Through the <script> Tag" works as a title. Would you also put it in navbar and on the page itself? Or keep headers inconsistent?

I'm not sure "through the script tag" also necessarily implies that you don't need to configure anything.